crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -67,9 +88,13 @@ pub async fn start_tx_checkpoint_commit_task<S>(
                 None,
                 &metrics,
                 &commit_notifier,
+                object_snapshot_backfill_mode,
             )
             .await;
         }
+        if latest_committed_cp + OBJECTS_SNAPSHOT_MAX_CHECKPOINT_LAG > latest_fn_cp {
```

Comment:
note that this is a one-way flip in case indexer falls behind and objects_snapshot table might be populated by both at the same time.